### PR TITLE
remove unnecessary class=external in web/html tree

### PR DESCRIPTION
**Pull request by peterbe**
The `class="external"` gets automatically injected at build-time anyway so no need to manually set it in the raw sources. 
Besides, this'll make more sense as we move towards something Markdown'ish.

### CODE REVIEW

**Comment by peterbe** [comment]
@nschonni Can you take a look with your hawk-like eyes?
I wrote a python script to make this PR.

File: files/en-us/web/html/element/keygen/index.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h2 id="Attributes">Attributes</h2>
...
+<p>For EC keys, the <code>keyparams</code> parameter specifies the name of the elliptic curve on which the key will be generated. It is normally a string from the table in <a href="http://mxr.mozilla.org/mozilla-central/source/security/manager/ssl/src/nsKeygenHandler.cpp?mark=179-185,187-206,208-227,229-256#177">nsKeygenHandler.cpp</a>. (Note that only a subset of the curves named there may actually be supported in any particular browser.) If the <code>keyparams</code> parameter string is not a recognized curve name string, then a curve is chosen according to the user's chosen key strength (low, medium, high), using the curve named "<code>secp384r1</code>" for high, and the curve named "<code>secp256r1</code>" for medium keys. (Note: choice of the number of key strengths, default values for each strength, and the UI by which the user is offered a choice, are outside of the scope of this specification.)</p>
```
Comment (nschonni) [inline files/en-us/web/html/element/keygen/index.html:76]: Should the other Mozilla domains get marked as `internal` as part of the build? I think a bunch of these MXR links are dead/moved though to firefoxsearch or something though

File: files/en-us/web/html/element/wbr/index.html [hunk 2]
```diff
@@ -63,7 +63,7 @@ <h2 id="Notes">Notes</h2>
 
 <h2 id="Example">Example</h2>
 
-<p><em><a class="external" href="https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/">The Yahoo Style Guide</a></em> recommends <a class="external" href="https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/editing/treat-abbreviations-capitalization-and-titles-consistently/website-names-and-addresses">breaking a URL <em>before</em> punctuation</a>, to avoid leaving a punctuation mark at the end of the line, which the reader might mistake for the end of the URL.</p>
+<p><em><a href="https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/">The Yahoo Style Guide</a></em> recommends <a href="https://web.archive.org/web/20121105171040/http://styleguide.yahoo.com/editing/treat-abbreviations-capitalization-and-titles-consistently/website-names-and-addresses">breaking a URL <em>before</em> punctuation</a>, to avoid leaving a punctuation mark at the end of the line, which the reader might mistake for the end of the URL.</p>
```
Comment (nschonni) [inline files/en-us/web/html/element/wbr/index.html:66]: Aside: Should there be a tracking issues for the various archive.org links to do something with them?

**Review by nschonni — APPROVED** [review]

File: files/en-us/web/html/element/keygen/index.html (hunk 1 above)
Comment (peterbe) [reply files/en-us/web/html/element/keygen/index.html:76]: No, it's also external.

File: files/en-us/web/html/element/wbr/index.html (hunk 2 above)
Comment (peterbe) [reply files/en-us/web/html/element/wbr/index.html:66]: ugh? Is there something wrong with them?

Comment (nschonni) [reply files/en-us/web/html/element/wbr/index.html:66]: It depends one what they're referencing really. EX: people used to abuse it as a CDN ☹️ , but if the pages aren't "live" anymore, they might not be a good example to reference in the MDN docs anymore

**Review by chrisdavidmills — APPROVED** [review]
I've checked a whole bunch of them, LGTM.

**Pull request merged**